packages/core/src/BaseSwitch/BaseSwitch.styles.tsx

```diff
@@ -63,20 +63,38 @@ export const { staticClasses, useClasses } = createClasses("HvBaseSwitch", {
     width: "40px",
     height: "32px",
     padding: 0,
+    [`&[data-size="lg"]`]: {
```

Comment:
there's no need for all this escaping
```suggestion
    "&[data-size=lg]": {
```